examples/ui/ui_texture_slice_flip_and_tile.rs

```diff
@@ -60,13 +60,11 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                 ([160., 160.], true, true),
             ] {
                 parent.spawn((
-                    NodeBundle {
-                        style: Style {
-                            width: Val::Px(width),
-                            height: Val::Px(height),
-                            ..default()
-                        },
-                        ..Default::default()
+                    Node::default(),
```

Comment:
This can be removed as UiImage requires Node. Lets move UiImage to the top of this declaration.

Reply:
Thanks! It's done.